foreman_discovery.gemspec

```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |s|
   s.summary = %q{MaaS Discovery Plugin for Foreman}
 
   s.add_dependency 'deface', '< 2.0'
+  s.add_dependency 'rails', '>= 4.1'
```

Comment:
We already have transitive dependency Discovery -> Core -> Rails, so I'd rather not introduce this one.